apps/bmslcd/bmsLcd.h

```diff
@@ -0,0 +1,25 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018 ARM Limited
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef BMS_LCD_H
+#define BMS_LCD_H
+
+#include "mbed.h"
+#include "TextLCD.h"
+
+class bmsLcd
+{
+    public:
+    bmsLcd(PinName rs, PinName e, PinName d4, PinName d5, PinName d6, PinName d7, LCDType type = TextLCD::LCD16x2);
+
+    void bmsLcd::writeToLcd(const char* firstLine, const char* secondLine)
```

Comment:
i dont really like the char *. I understand that it is bc you likely need to send a contiguous data structure, which std::strings are not. As a solution, I recommend using std::string throught your api and within your function calls convert cpp string to c string with .c_str()